src/AppBundle/Resources/views/restaurant/product.html.twig

```diff
@@ -41,6 +41,20 @@
 
   <hr>
 
+  <div class="row">
+    <div class="col-sm-6">
+      {{ form_row(form.refundableContainersEnabled) }}
+      {% if form.refundableContainersEnabled.vars.data == true %}
+      <span id="refund-unit-enabled">
+        {{ form_row(form.refundableContainersUnit, {'attr': {'class': 'form-control input-number'}}) }}
+      </span>
+      {% else %}
+        <span id="refund-unit-enabled" style="display: none">
```

Comment:
Unclosed `<span>` tag here. Moreover, I think it should be a `<div>` (because `<span>` should only contain [inline elements](https://developer.mozilla.org/fr/docs/Web/HTML/%C3%89l%C3%A9ments_en_ligne))